src/modules/user/routes.js

```diff
@@ -65,8 +65,12 @@ usersRouter.post(
 
 // GET /users
 usersRouter.get("/", async (req, res) => {
-  const users = await databasePrisma.user.findMany();
-  res.json(users);
+  try {
+    const users = await databasePrisma.user.findMany();
```

Comment:
we need to remove the salt and password from the users in the array something like this should do the trick.
```js
  users.forEach((user) => {
    delete user.password;
    delete user.salt;
  });
  ```